File: Python/Linear_Search.py



def LinearSearch(DataStore, size_of_store,search):
    countData = 0
    for x in range(0,size_of_store):
        if(DataStore[x]==search):
            countData += countData
    
    if(countData !=0):
        print(f"{search} Data is found {countData} times")
    else:
        print(f"{search} Data is not found")




DataStore = [20,10,30,5,9,7]
size_of_store = len(DataStore)
search = input("Enter search value : ")
LinearSearch(DataStore,size_of_store,search)

File: C++/Linear_Search.cpp

#include<iostream>
using namespace std;


void LinearSearch(int DataStore[], int size_of_Store, int search){
 
 int countData = 0;

    for(int a=0; a<size_of_Store; a++){
        if(DataStore[a]==search){
            countData = countData+1;
        }
    }

    if(countData !=0){
        cout<<search<<" Data Found "<< countData<<" times"<<endl;
    }
    else{
        cout<<search<<" Data not Found !"<<endl;
    }

}



int main(){

int DataStore[] = {20,10,30,5,9,7};
int size_of_Store = sizeof(DataStore)/sizeof(DataStore[0]);

cout<<"Enter search Value : ";
int search;
cin>>search;

LinearSearch(DataStore,size_of_Store,search);

    return 0;
}